### PR TITLE
docs: update README

### PR DESCRIPTION
**Pull request by vas3k**
Added more structure and information for beginners

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,118 @@
...
+- _Unique._ Do not duplicate existing opened issues.
+- _Scoped to a Single Bug._ One bug per report.
+
+Official team email for direct inquiries: signer@parity.io
+
+# Contributing
+
+Our contribution guidelines are still in development. Until then, you're welcome to participate in discussions and send PRs with small bugfixes, we'd love it. Each PR must be reviewed by at least two project maintainers.
+
+# Links
```
Comment (goldsteinsveta) [inline README.md:108]: I'd put links right after Introduction

**Review by goldsteinsveta — COMMENTED** [review]
thank you for doing this!

File: README.md [hunk 2]
```diff
@@ -1,44 +1,117 @@
...
+☝️ **Disabling the mobile phone's networking abilities is a requirement for the app to be used as intended, check our [wiki](https://paritytech.github.io/parity-signer/about/Security-And-Privacy.html) for more details.**
 
-https://paritytech.github.io/parity-signer/index.html
+Any data transfer from or to the app happens using QR code. By doing so, the most sensitive piece of information, the private keys, will never leave the phone. The Parity Signer mobile app can be used to store any Substrate account, this includes Polkadot (DOT) and Kusama (KSM) networks.
 
-==========================================================
+**Available for both iOS and Android.**
 
-# !!! Parity Signer was refactored from the ground up !!!
+![](docs/src/res/screens-for-readme.png)
```
Comment (goldsteinsveta) [inline README.md:31]: screens-for-readme

could you try these? these are a bi calmer and - bonus - will work for dark and light gh ui

**Review by goldsteinsveta — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (vas3k) [reply README.md:31]: Sure, thanks for making it pretty :)

**Pull request merged**